### PR TITLE
[ace] Add support for uwp

### PR DESCRIPTION
**Pull request by jwillemsen**
**Describe the pull request**

- Which triplets are supported/not supported? Have you updated the CI baseline?

ace:arm-uwp and ace:x64-uwp, should work now

### CODE REVIEW

File: ports/ace/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: ace
-Version: 6.5.9
+Version: 6.5.9-1
 Homepage: https://www.dre.vanderbilt.edu/~schmidt/ACE.html
 Description: The ADAPTIVE Communication Environment
 Supports: !uwp
```
Comment (NancyLi1013) [inline ports/ace/CONTROL:5]:
```suggestion
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by jwillemsen** [comment]
Contains the changes from #11369 which I would expect to be merged first

**Comment by jwillemsen** [comment]
Looks arm64_windows failures are not related to my changes

**Comment by NancyLi1013** [comment]
Yes, this failure is not related with this PR. 
You can ignore this. We will resolve this.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
All features have passed with the following triplets:
- x86-windows
- x64-windows
- x64-windows-static

**Comment by NancyLi1013** [comment]
This PR should be merged after #11369.

**Review by dan-shaw — APPROVED** [review]

**Comment by dan-shaw** [comment]
LGTM, can you fix the conflicts from your PR that was just merged?

**Comment by jwillemsen** [comment]
Resolved merge conflicts, thanks for merging the first PR!

**Pull request merged**